zuul.d/arista-eos-jobs.yaml

```diff
@@ -69,213 +48,92 @@
     vars:
       ansible_collections_repo: github.com/ansible-collections/arista.eos
 
-- job:
-    name: ansible-test-network-integration-eos-httpapi
-    parent: ansible-test-network-integration-eos
-    abstract: true
-    vars:
-      ansible_test_collections: true
-      ansible_test_skip_tags: local,network_cli
-
-- job:
-    name: ansible-test-network-integration-eos-network_cli
-    parent: ansible-test-network-integration-eos
-    abstract: true
-    vars:
-      ansible_test_collections: true
-      ansible_test_skip_tags: local,httpapi
-
-- job:
-    name: ansible-test-network-integration-eos-python27
-    parent: ansible-test-network-integration-eos
-    nodeset: eos-4.24.6-python27
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-eos-httpapi-python27
-    parent: ansible-test-network-integration-eos-httpapi
-    nodeset: eos-4.24.6-python27
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-eos-network_cli-python27
-    parent: ansible-test-network-integration-eos-network_cli
-    nodeset: eos-4.24.6-python27
-    vars:
-      ansible_test_python: 2.7
+# =============================================================================
 
 - job:
-    name: ansible-test-network-integration-eos-httpapi-python27-stable211
-    parent: ansible-test-network-integration-eos-httpapi
-    nodeset: eos-4.24.6-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
-    vars:
-      ansible_test_python: 2.7
+    name: network-ee-integration-tests-eos
+    parent: ansible-network-eos-appliance
 
 - job:
-    name: ansible-test-network-integration-eos-network_cli-python27-stable211
-    parent: ansible-test-network-integration-eos-network_cli
-    nodeset: eos-4.24.6-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
+    name: network-ee-integration-tests-eos
+    parent: network-ee-integration-tests
+    nodeset: eos-4.24.6-python38
     vars:
-      ansible_test_python: 2.7
-      ansible_test_split_in: 2
+      ansible_test_integration_targets: "eos_.*"
+      ansible_test_fips_mode: true
+      ansible_test_skip_tags: local,httpapi
 
 - job:
-    name: ansible-test-network-integration-eos-network_cli-python27-stable211-scenario01
-    parent: ansible-test-network-integration-eos-network_cli-python27-stable211
+    name: network-ee-integration-tests-eos-libssh-scenario01
+    parent: network-ee-integration-tests-eos
     vars:
+      ansible_test_network_cli_ssh_type: libssh
       ansible_test_do_number: 1
```

Comment:
we don't support ansible_test_do_number flag, so this should be removed.  Lets see how fast / slow the jobs run.